packages/core/workers/src/WorkerFarm.js

```diff
@@ -389,6 +389,10 @@ export default class WorkerFarm extends EventEmitter {
     return handle;
   }
 
+  clearConfigCache(): void {
+    loadConfig.clear();
```

Comment:
This will only clear the cache on the main thread. We need to send this to all of the workers. See above comment.